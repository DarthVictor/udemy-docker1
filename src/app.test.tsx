import { expect, it } from 'vitest'

import { render } from '@testing-library/preact';

import { App } from './app.tsx'

it('App should render', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
