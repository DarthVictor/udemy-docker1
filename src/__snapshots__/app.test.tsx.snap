// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App should render 1`] = `
<DocumentFragment>
  <div>
    <a
      href="https://vite.dev"
      target="_blank"
    >
      <img
        alt="Vite logo"
        class="logo"
        src="/vite.svg"
      />
    </a>
    <a
      href="https://preactjs.com"
      target="_blank"
    >
      <img
        alt="Preact logo"
        class="logo preact"
        src="/src/assets/preact.svg"
      />
    </a>
  </div>
  <h1>
    Vite + Preact
  </h1>
  <p>
    Hello there!
  </p>
  <p
    class="read-the-docs"
  >
    Click on the Vite and Preact logos to learn more
  </p>
</DocumentFragment>
`;

exports[`App should render 2`] = `
<DocumentFragment>
  <div>
    <a
      href="https://vite.dev"
      target="_blank"
    >
      <img
        alt="Vite logo"
        class="logo"
        src="/vite.svg"
      />
    </a>
    <a
      href="https://preactjs.com"
      target="_blank"
    >
      <img
        alt="Preact logo"
        class="logo preact"
        src="/src/assets/preact.svg"
      />
    </a>
  </div>
  <h1>
    Vite + Preact
  </h1>
  <p>
    Hello there!
  </p>
  <p
    class="read-the-docs"
  >
    Click on the Vite and Preact logos to learn more
  </p>
</DocumentFragment>
`;
